usecases/config/config_handler.go

```diff
@@ -40,13 +40,6 @@ import (
 // spec only needs to be parsed once.
 var ServerVersion string
```

Comment:
Now I don't know why we need this?

How is it different than `Version`?

Reply:
we resolved this during our huddle :)